graph/types.go

```diff
@@ -113,6 +117,10 @@ func NewNodeExplicit(id, namespace, workload, app, version, service, nodeType, g
 		app = ""
 		workload = ""
 		version = ""
+
+		if service == passthroughCluster {
```

Comment:
Oh! So, it's just the service name the one that triggers the flag.
This turns these changes quite straighforward :)

Reply:
Yes, as far as I can see this is just a special-case service name used as a "default service entry" when external traffic is routed out of the cluster.